src/main/java/no/fint/portal/model/asset/AssetService.java

```diff
@@ -33,8 +35,15 @@ private boolean createAsset(Asset asset, Organisation organisation, boolean prim
         return ldapService.createEntry(asset);
     }
 
-    public boolean addAsset(Asset asset, Organisation organisation) {
-        return createAsset(asset, organisation, false);
+    public void addAssetWithPrimary(Asset asset, Organisation organisation) {
```

Comment:
What do you mean by addAssetWithPrimary?

Reply:
Sorry for that. Probably not the best naming. But I did want to change the name as the functionality has changed. This method now combines assetId with the primary asset's assetId. 
What would be a better name? I notice that we have a addPrimaryAsset. How about addChildAsset or addSubAsset?

Reply:
What about `addChildAsset` @fsjovatsen?